src/fr/composants/recherche/case-dusage.md

```diff
@@ -6,26 +6,30 @@ eleventyNavigation:
   title: Recherche
   locale: fr
   parent: componentsFR
-  otherNames: barre de recherche, champ de recherche du site, boîte de recherche, correspondance.
+  otherNames: barre de recherche, boîte de recherche sur le site, champ de recherche.
   description: Un espace où entrer des mots clés pour chercher des renseignements pertinents.
   thumbnail: /images/common/components/preview-search.svg
   alt: Le composant Recherche affiche un champ de saisie de recherche vide avec un curseur à l'intérieur. À la fin du champ de saisie se trouve un rectangle bleu foncé comprenant un bouton avec une icône de loupe.
-  state: coming-soon
+  tag: De base
+  state: published
 translationKey: 'search'
 tags: ['searchFR', 'usage']
 permalink: /fr/composants/recherche/
-# date: "git Last Modified"
+date: "git Last Modified"
 ---
 
-## Problems component solves
+## Cas d’utilisation du composant recherche
 
-Use a Recherche
+Utilisez le composant recherche pour aider les personnes utilisant votre site Web à trouver des informations :
 
--
+- Indexées sur Canada.ca, dans tout le contenu du gouvernement du Canada, ou encore au niveau de l’organisme ou du programme.
+- À partir d’un sous-ensemble de contenu publié, en utilisant votre propre ensemble de données.
 
 <article class="bg-full-width bg-primary text-light pt-500 pb-400 my-500">
   <h2 class="mt-0 mb-400">Composants connexes</h2>
 
-<a href="" class="link-light">component</a>
+<a href="{{ links.header }}" class="link-light">En-tête :</a> utilisé pour le placement de l'en-tête portant l'image de marque du gouvernement du Canada.
```

Comment:
I'm sorry this is a very nit-picky comment but can you please move the space and colon out of the link? :D :D :D It looks a bit off to have the spacing between the word and the colon underlined.